testing/spectest/shared/phase0/epoch_processing/registry_updates.go

```diff
@@ -27,8 +27,6 @@ func RunRegistryUpdatesTests(t *testing.T, config string) {
 	}
 }
 
-func processRegistryUpdatesWrapper(t *testing.T, state state.BeaconState) (state.BeaconState, error) {
-	state, err := epoch.ProcessRegistryUpdates(context.Background(), state)
-	require.NoError(t, err, "Could not process registry updates")
-	return state, nil
+func processRegistryUpdatesWrapper(_ *testing.T, state state.BeaconState) (state.BeaconState, error) {
+	return epoch.ProcessRegistryUpdates(context.Background(), state)
```

Comment:
Spec test requires this test to fail. In this case, we don't want to error out, and we want to return the error to the caller and the caller can decide whether to pass or fail